src/ucp/tag/tag_send.c

```diff
@@ -36,8 +39,19 @@ static ucs_status_t ucp_tag_req_start(ucp_request_t *req, size_t count,
         req->send.state.dt.iov.iov_offset    = 0;
         req->send.state.dt.iov.iovcnt        = count;
         flag_iov_single                      = (count <= config->am.max_iovcnt);
+        if (0 == count || (zcopy_thresh_arr[0] != UCS_CONFIG_MEMUNITS_AUTO)) {
```

Comment:
why is this case needed? if (count==0) if would say zcopy_thresh=SIZE_MAX (disable zcopy)